packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js

```diff
@@ -24,15 +25,25 @@ const ISSUE_BODY = `
 [NOTE]: # (Please provide specifics of the local error logs, Browser Dev Tools console, etc. if appropriate and possible.)
 `;
 
-const CreateIssueButton = ({ pageTitle, ...props }) => {
+const CreateIssueButton = ({
+  instrumentation,
+  pageTitle,
+  onClick,
+  ...props
+}) => {
   const { t } = useThemeTranslation();
+  const handleClick = useInstrumentedHandler(onClick, {
+    actionName: 'createAnIssue_click',
+    component: instrumentation?.component,
```

Comment:
Out of curiosity, is there any instances where we would _not_ have `instrumentation` or `instrumentation.component`? Both are marked as `isRequired`.

Reply:
I made it required because I think we should really encourage instrumentation on this component. Regardless, this component will be instrumented 100% of the time, its just a matter of whether the `component` property is set or not. Figured its best to set that, therefore I made it required so we could best track where this click originated from.